synapse/rest/media/v1/preview_url_resource.py

```diff
@@ -402,7 +403,10 @@ async def _download_url(self, url: str, output_stream: BinaryIO) -> DownloadResu
                 url,
                 output_stream=output_stream,
                 max_size=self.max_spider_size,
-                headers={"Accept-Language": self.url_preview_accept_language},
+                headers={
+                    b"Accept-Language": self.url_preview_accept_language,
+                    b"User-Agent": ["Synapse (bot)"],
```

Comment:
```suggestion
                    "Accept-Language": self.url_preview_accept_language,
                    "User-Agent": ["Synapse (bot)"],
```

For consistency.

Reply:
Oh yes, very good point.

Reply:
in SimpleHttpClient the headers are all specified as bytes strings.  If we pass this map with a regular string key, python will not overwrite the default user-agent key and put both 'User-Agent' and b'User-agent' both in the headers map.  Since SimpleHttpClient uses bytes strings for all headers throughtout client.py, I believe that this is the correct behavior.

I will update the agent string as suggested